### PR TITLE
[11.0] [MIG] project_description

### PR DESCRIPTION
**Pull request by emagdalenaC2i**
simple migration (despite GitHub) to Odoo 11

### CODE REVIEW

**Comment by emagdalenaC2i** [comment]
@pedrobaeza , @astirpe and @Yajo could you please review this PR?

File: project_description/README.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (elicoidal) [inline project_description/README.rst]: svg -> png

File: project_description/README.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+This module extends the functionality of projects to support setting a
+description for each project and allow you to search by that description.
+
+Usage
+=====
+
+To set a description for a project, you need to:
+
+* Go to *Project > Project > Projects*.
```
Comment (elicoidal) [inline project_description/README.rst]: better use enumerated list for procedures `#. `

File: project_description/README.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* Edit or create one.
+* You will see a new *Description* tab, with an HTML area to set the project's
```
Comment (elicoidal) [inline project_description/README.rst]: a screenshot is welcome (nice to have)

File: project_description/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
```
Comment (elicoidal) [inline project_description/__init__.py]: copyright notice not necessary in `__init__.py`

**Review by elicoidal — APPROVED** [review]
some suggestions.
LGTM

File: project_description/__manifest__.py [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
 {
     "name": "Project Description",
     "summary": "Add a description to projects",
-    "version": "10.0.1.0.0",
-    "category": "Project Management",
-    "website": "https://www.tecnativa.com",
+    "version": "11.0.1.0.0",
+    "category": "Project",
+    "website": "https://github.com/OCA/project",
     "author": "Antiun Ingeniería S.L., "
```
Comment (pedrobaeza) [inline project_description/__manifest__.py]: Please remove this old reference, as the new company is Tecnativa

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: project_description/__manifest__.py (hunk 5 above)
Comment (emagdalenaC2i) [reply project_description/__manifest__.py]: You mean Antiun Ingeniería (http://antiun.com/) is no longer exist and it is now Tecnativa?
If so, should I change it also in models/__init__.py and models/project.py?

Also, should I change the contributors in the README.rst to:
* Tecnativa (https://www.tecnativa.com): Rafael Blasco <rafael.blasco@tecnativa.com>
* Tecnativa (https://www.tecnativa.com): Jairo Llopis <jairo.llopis@tecnativa.com>

File: project_description/__init__.py (hunk 4 above)
Comment (emagdalenaC2i) [reply project_description/__init__.py]: @elicoidal and @pedrobaeza Should I remove this also in models/init.py?

File: project_description/__manifest__.py (hunk 5 above)
Comment (pedrobaeza) [reply project_description/__manifest__.py]: Well, Antiun still exists, but Antiun's Odoo division is now part of Tecnativa.

File: project_description/__init__.py (hunk 4 above)
Comment (pedrobaeza) [reply project_description/__init__.py]: Yeah, I think so. `__init__.py` files are not code itself, just a tool for loading the actual code.

File: project_description/__manifest__.py (hunk 5 above)
Comment (emagdalenaC2i) [reply project_description/__manifest__.py]: so?

Comment (pedrobaeza) [reply project_description/__manifest__.py]: Remove it and leave Tecnativa, please

Comment (emagdalenaC2i) [reply project_description/__manifest__.py]: ok

**Comment by emagdalenaC2i** [comment]
Done.
I will try to rebase later

File: project_description/README.rst [hunk 6]
```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
```
Comment (astirpe) [inline project_description/README.rst]: Update the link according to the [latest template](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)

File: project_description/README.rst [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+Usage
+=====
+
+To set a description for a project, you need to:
+
+#. Go to *Project > Project > Projects*.
+#. Edit or create one.
+#. You will see a new *Description* tab, with an HTML area to set the project's
+  description.
```
Comment (astirpe) [inline project_description/README.rst]: This line causes the runbot failure. Write only one single line for each item of the list.

File: project_description/README.rst [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/11.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/project/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
```
Comment (astirpe) [inline project_description/README.rst]: Update the text according to [latest template](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)

File: project_description/models/project.py [hunk 9]
```diff
@@ -0,0 +1,10 @@
+# © 2015-2017 Tecnativa - Jairo Llopis
```
Comment (astirpe) [inline project_description/models/project.py]: Replace `©` with `Copyright`

File: project_description/views/project_view.xml [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<odoo>
+
+    <record id="project_form" model="ir.ui.view">
+        <field name="name">Project description</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.edit_project"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='misc']/.." position="before">
```
Comment (astirpe) [inline project_description/views/project_view.xml]: You should simply write:

```
<xpath expr="//page[@name='settings']" position="before">
```

**Review by astirpe — CHANGES_REQUESTED** [review]

File: project_description/README.rst (hunk 7 above)
Comment (pedrobaeza) [reply project_description/README.rst]: Or you can indent this line with 3 spaces at the beginning

**Review by yajo — APPROVED** [review]
Seems OK, once handled all others' comments.

**Comment by emagdalenaC2i** [comment]
Done.
But it would be nice if someone show me how to squash all my commits ;-)

**Comment by pedrobaeza** [comment]
Check https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests for a little reference

**Comment by emagdalenaC2i** [comment]
I did, but when I run "$ git rebase -i origin/11" I get
_fatal: Needed a single revision
invalid upstream origin/11_

**Comment by pedrobaeza** [comment]
It's `git rebase -i origin/11.0`

**Comment by carlosdauden** [comment]
Try with origin/11.0 😉

**Comment by emagdalenaC2i** [comment]
Thanks
I hate when that stupid mistakes happen

**Review by astirpe — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]
Merge after rebase

**Comment by elicoidal** [comment]
@pedrobaeza I think your remarks were attended

**Comment by emagdalenaC2i** [comment]
@elicoidal yes, but I am trying to squash all my commits

**Comment by emagdalenaC2i** [comment]
It should be ok now
Thank you all for your patience and support

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**